app/js/adminPage.js

```diff
@@ -9,17 +15,17 @@ populateHandlebars('#testAllocated', 'js/templates/testAllocatedFilter.hbs', 'te
  * @returns A promise containing the response, which includes the boolean success property.
  */
 async function saveNewUser(user) {
-    let baseUrl = getBaseUrl()
-    let formData = jsonToFormData(user) // API does not work with JSON - needs form data
+    let baseUrl = getBaseUrl();
+    let formData = jsonToFormData(user); // API does not work with JSON - needs form data
     let apiData = await fetch(
         baseUrl + 'user',
         {
             method: 'post',
             body: formData
         }
-    )
+    );
 
-    apiData = await apiData.json()
+    apiData = await apiData.json();
     return apiData
```

Comment:
add ; to this?

Reply:
there's a few missing ; from the adminPage.js file.